lib/uplink/project.go

```diff
@@ -147,20 +148,37 @@ func (p *Project) GetBucketInfo(ctx context.Context, bucket string) (b storj.Buc
 
 // OpenBucket returns a Bucket handle with the given EncryptionAccess
 // information.
-func (p *Project) OpenBucket(ctx context.Context, bucketName string, access *EncryptionAccess) (b *Bucket, err error) {
+func (p *Project) OpenBucket(ctx context.Context, bucketName string, accesses ...*EncryptionAccess) (b *Bucket, err error) {
 	defer mon.Task()(&ctx)(&err)
 
 	bucketInfo, cfg, err := p.GetBucketInfo(ctx, bucketName)
 	if err != nil {
 		return nil, err
 	}
 
-	if access == nil || access.Key == (storj.Key{}) {
-		return nil, Error.New("No encryption key chosen")
+	// Construct a searcher from the accesses. This allows finding the appropriate key for
+	// any path in the bucket.
+	if len(accesses) != 1 {
```

Comment:
Shouldn't this be `if len(accesses) > 1`?

Reply:
Yep.